invenio_rdm_records/resources/resources.py

```diff
@@ -158,13 +154,42 @@ def pids_discard(self):
 
         return item.to_dict(), 200
 
-    def record_communities_add(self):
+
+class RDMRecordCommunitiesResource(ErrorHandlersMixin, Resource):
+    """Record communities resource."""
+
+    def __init__(self, config, service):
+        """Constructor."""
+        super().__init__(config)
+        self.service = service
+
+    def create_url_rules(self):
+        """Create the URL rules for the record resource."""
+        routes = self.config.routes
+        url_rules = [
+            route("POST", routes["item-communities"], self.add),
+            route("DELETE", routes["item-communities"], self.remove),
+        ]
+        return url_rules
+
+    def add(self):
         """Add communities to a record."""
         return {}, 200
```

Comment:
Shall we return something to clarify that this is not implemented yet? I know we are in dev, but still it might cause misunderstandings.